src/app/admin/websiteDataTable/FetchWebsiteData.tsx

```diff
@@ -0,0 +1,51 @@
+import supabase from "@/supabaseClient";
+import { DatabaseError } from "@/app/errorClasses";
+import { WebsiteDataRow } from "./WebsiteDataTable";
+import { Tables } from "@/databaseTypesFile";
+import { logErrorReturnLogId } from "@/logger/logger";
+
+type DbWebsiteData = Tables<"website_data">;
+
+const getReadableName = (name: string): string =>
+    name
+        .split("_")
+        .map((word) => `${word[0].toUpperCase()}${word.slice(1)}`)
+        .join(" ");
+
+const getUnreadableName = (name: string): string =>
+    name
+        .split(" ")
+        .map((word) => word.toLowerCase())
+        .join("_");
```

Comment:
I don't think these are strictly reverse of each other.

If we have a column called `A`, this becomes `A` through `getReadableName` and `a` through `getUnreadableName`.

All this to say, maybe `WebsiteDataRow` should just have an additional field `dbName` or something to just keep track of the original db value so it would always always work.

Reply:
Ah I see. Good solution